wolfSSL/src/chapter02.md

```diff
@@ -628,6 +1002,86 @@ Enables Blake2s algorithm support
 
 Enables Signaling Cipher Suite Value(SCSV) support on the server side. This handles the cipher suite 0x56 0x00 sent from a client to signal that no downgrade of TLS version should be allowed.
 
+#### HAVE_AEAD
+
+Implements the use of AEAD and is required for TLS 1.3.
+
+#### HAVE_AES_CBC
+
+Enable option for AES.
+
+#### HAVE_AES_DECRYPT
+
+This setting controls whether the decrypt direction of the various AES modes is supported.
+
+#### HAVE_ALPN
+
+Crypto enables the option for application-layer protocol negotiation.
+
+#### HAVE_CAVIUM_OCTEON_SYNC
+
+By default, the OCTEON's global variables are all thread local. This tag allows them to be shared between threads.
+
+#### HAVE_CERTIFICATE_STATUS_REQUEST
+
+Used for Certificate revocation as a cert status request feature.
+
+#### HAVE_CERTIFICATE_STATUS_REQUEST_V2
+
+Used for Certificate revocation as a cert status request feature.
+
+#### HAVE_IO_TIMEOUT
+
+Certificate revocation. IO options enable support for connect timeout, but the default is off.
+
+#### HAVE_CURL
+
+Used for building a subset of the wolfSSL library when linking with cURL.
+
+#### HAVE_CURVE448
+
+Define for Curve448 support. Additional macro settings can be changed. The default is to enable shared secret, key export, and import.
+
+#### HAVE_DANE
+
+This macro is a protocol that uses DNSSEC(domain lookup with crypto signatures) to allow certificates to be validated without depending certificate authority. This can provide a stronger garuntee of legitamicy. To verify Raw Public Key cert, DANE(RFC6698) add macro HAVE_DANE for DANE authentication.
```

Comment:
DANE is not supported yet with Raw Public Keys. This option is only supported with HAVE_RPK (Raw Public Keys) and is a placeholder for when it might be added in the future.